### ISSUE TITLE
posting from url connection error

### ISSUE THREAD

**Issue by horusra** [ccd0/4chan-x#2079]
when using a url of a file to post with, 4chan-x will try to upload but constantly give `Connection error while posting. [More info]`
![image](https://user-images.githubusercontent.com/17074394/46264896-d69ce800-c521-11e8-8d19-91c1f059de82.png)
Using this feature, linking to a webm (from another board on 4chan) and then trying to post, always results in the error mentioned above. I tried disabling my addons (umatrix, ublock etc) but the issue remains,

**Comment by ccd0** [comment]
Does the same thing happen if you download the file and select it from the file input field?

**Comment by horusra** [comment]
no, downloading the file and choosing it that way posts just fine. it only happens when using the post from url feature.
